src/clj/fluree/db/ledger.cljc

```diff
@@ -163,6 +165,16 @@
     (subs ledger-alias 1)
     ledger-alias))
 
+(defn publish-commit
```

Comment:
This function operates on a connection, so it should be in the connection namespace. Ledgers shouldn't reach in to connections, and they should not know anything about name services, or anything else attached to a connection besides the ledger itself. Publishing commits is not and should not be the concern of ledger objects.

Reply:
Good catch, I've refactored to do the correct thing instead of the expedient thing :+1: